dev-docs/modules/userid-submodules/utiq.md

```diff
@@ -1,27 +1,27 @@
 ---
 layout: userid
-title: Trustpid
-description: Trustpid User ID sub-module
-useridmodule: trustpidSystem
+title: Utiq
+description: Utiq User ID sub-module
+useridmodule: utiqSystem
 ---
 
 
-Trustpid generates unique tokens, enabling improved efficiency in programmatic advertising while safeguarding transparency and control for end customers via `trustpid.com`. A website visitor’s Trustpid is generated based on network identifiers provided by network operators and requires explicit user consent.
+Utiq generates unique tokens, enabling improved efficiency in programmatic advertising while safeguarding transparency and control for end customers via `utiq.com`. A website visitor’s Utiq is generated based on network identifiers provided by network operators and requires explicit user consent.
```

Comment:
@jkthomas please correct utiq.com to consenthub.utiq.com.
@muuki88 FYI small mistake we need to change.